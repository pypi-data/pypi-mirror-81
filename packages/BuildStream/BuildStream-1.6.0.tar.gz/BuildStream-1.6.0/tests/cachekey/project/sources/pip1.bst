kind: import

sources:
- kind: git
  url: https://example.com/foo/foobar.git
  ref: b99955530263172ed1beae52aed7a33885ef781f
- kind: pip
  url: https://pypi.example.com/simple
  packages:
  - horses
  - ponies
  ref: 'horses==0.0.1\nponies==0.0.2'
