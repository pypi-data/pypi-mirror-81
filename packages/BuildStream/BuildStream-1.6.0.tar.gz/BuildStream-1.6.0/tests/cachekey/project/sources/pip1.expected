492115a2d3a54eb5fd4a301f1cdfd328b2d3dd918e709386cea8133f8cac53eb